README.md

```diff
@@ -209,6 +216,18 @@ This will create a folder containing the basic structure for a kody. You can sta
 npm i
 ```
 
+To address issues that do not require attention, run:
```

Comment:
These issues are parts of packages that we need have as seperate package. Thank for reminding me about the issues that require attention. I'll open a new issue to have them fixed. For the now, I believe these should not be part of the desciption